props/saber_fett263_buttons.h

```diff
@@ -45,14 +85,48 @@ SaberFett263Buttons() : PropBase() {}
   bool swinging_ = false;
   void Loop() override {
     PropBase::Loop();
+    // Swing On gesture control this portion allows fine tuning of speed needed to ignite
+    if(!SaberBase::IsOn()) {
+      if (millis() - saber_off_time_ < MOTION_TIMEOUT) {
+        SaberBase::RequestMotion();
+      // Edit '250' value in line below to change swing on sensitivity, 250 ~ 400 work best in testing
+        if (!swinging_ && fusor.swing_speed() > 250) {
+          swinging_ = true;
+          Event(BUTTON_NONE, EVENT_SWING);
+        }
+        if (swinging_ && fusor.swing_speed() < 100) {
+          swinging_ = false;
+        }
+      }
+    } 
     if (!swinging_ && fusor.swing_speed() > 250) {
       swinging_ = true;
       Event(BUTTON_NONE, EVENT_SWING);
-    }
+    }	  
+    if (auto_lockup_on_ && 
+        !swinging_ && 
+        fusor.swing_speed() > 120 && 
+        millis() - clash_impact_millis_ > FETT263_LOCKUP_DELAY) {
+      if (SaberBase::Lockup()) {
```

Comment:
I think we should just merge this with the surrounding if, so like:

if (auto_lockup_on &&
     !swinging_ &&
     fusor.swing_speed() > 120 &&
     millis() - clash_impact_millis_ > FETT264_LOCKUP_DELAY &&
     SaberBase::Lockup()) {
  SaberBase::DoEndLockup();
  SaberBase::SetLockup(SaberBase::LOCKUP_NONE);
  auto_lockup_on_ = false;
}

and the same for the auto melt below...